docs/architecture/architecture-overview.adoc

```diff
@@ -0,0 +1,260 @@
+= Mobile Core (MC) Architecture Overview
+:toc:
+
+.Target Audience
+****
+The target audience for this document is developers who want a better understanding of what Mobile Core does, what value it brings and if they were to build a Mobile Service, how would they get MC to discover it and integrate with it.
+****
+
+== Terminology
+
+****
+*Mobile Service*;
+A service which delivers functionality that can be used as part of a Mobile solution. These services are offered through the OpenShift Service Catalog.
+
+*Mobile Client App*;
+This is the Mobile Developers client application which will run on the Mobile device. It will interact with the Mobile Services which are provisioned via the SDKs.
+
+*Mobile App Representation*;
+A Mobile App Representation is a set of basic properties for the App. It is not a running process. Currently its a visual represenation of an App. It is a resource that represents your mobile client app as part of the OpenShift UI. It gives the context and information needed to show you relevant information around your particular mobile runtime as well as allowing the setup of different kind of mobile client builds required.
```

Comment:
I'm presuming this is the term that replaces 'mobile app client' ?

MAR defines the mobile application on OpenShift (eg MobileHR). You can access it using mobile cli to set and retrieve configuration properties (eg auth configuration). For example, to view all the configuration and context associated with your app:

mobile --namespace=MobileHR get clientconfig

Reply:
Am going to leave the definition as is Paul for now. I think it can be defined more clearly when the development has progressed further.

Reply:
sure, just setting expectation for what we want to publish